lib/deploy.ts

```diff
@@ -1,4 +1,11 @@
 import { BaseConfig } from "./config";
+import { uploadAndPinJson } from '@archetype-org/ribbit';
```

Comment:
What if we moved all of this "deploy_with_metadata" to archetype library?

The attest command can be a direct mapping to ribbit and your contract.

-- although it appears that repo private. We're not sharing builds with a private api.